### ISSUE TITLE
flax error: no working directory specified

### ISSUE THREAD

**Issue by vnpower** [google-deepmind/alphageometry#53]
I run all the command at 1st time successfully.
When I run below command for 2nd time:
```
python -m alphageometry \
--alsologtostderr \
--problems_file=$(pwd)/examples.txt \
--problem_name=orthocenter \
--mode=alphageometry \
"${DDAR_ARGS[@]}" \
"${SEARCH_ARGS[@]}" \
"${LM_ARGS[@]}"
```
Then error occur:

> I0124 21:39:21.281035 139724541399040 nn_components.py:261] mlp: residual
> I0124 21:39:21.281769 139724541399040 transformer_base.py:450] tbase: ys = Traced<ShapedArray(bfloat16[2,1,1024])>with<DynamicJaxprTrace(level=2/0)>
> I0124 21:39:21.281977 139724541399040 decoder_stack.py:344] dstack: Final layernorm.
> I0124 21:39:21.286813 139724541399040 decoder_stack.py:365] dstack: logits = Traced<ShapedArray(float32[2,1,1024])>with<DynamicJaxprTrace(level=2/0)>
> I0124 21:39:45.593152 139724541399040 optimizer_config.py:74] Using Flax Adafactor Optimizer. lr=1.000000, b1=0.900000
> /home/vikisu/Desktop/alphageometry/lib/python3.10/site-packages/flax/optim/base.py:49: DeprecationWarning: Use `optax` instead of `flax.optim`. Refer to the update guide https://flax.readthedocs.io/en/latest/howtos/optax_update_guide.html for detailed instructions.
>   warnings.warn(
> I0124 21:39:47.651232 139724541399040 training_loop.py:409] No working directory specified.
> I0124 21:39:47.652373 139724541399040 training_loop.py:431] Loading pre-trained model from ag_ckpt_vocab:
> I0124 21:39:47.762608 139724541399040 checkpoints.py:429] Restoring checkpoint from ag_ckpt_vocab/checkpoint_10999999
> Killed
> 

Please help about this

**Comment by EricChen0803** [comment]
Same here, have you solved this?
